spec/system/grant_submissions/grant_submission_submission_applicants_spec.rb

```diff
@@ -513,14 +517,18 @@
               expect(page).not_to have_text(draft_applicant_02.email)
             end
 
-            scenario 'can add an applicant' do
+            scenario 'can add an applicant only once' do
               new_applicant
 
               expect(page).not_to have_text(new_applicant.email)
               find_field(id: 'grant_submission_submission_applicant_applicant_email').set(new_applicant.email)
               click_button 'Look Up'
               expect(page).to have_text("#{new_applicant.first_name} #{new_applicant.last_name} was added as applicant on #{draft_submission.title}.")
               expect(page).to have_text(new_applicant.email)
+
+              find_field(id: 'grant_submission_submission_applicant_applicant_email').set(new_applicant.email)
+              click_button 'Look Up'
+              expect(page).to have_text("Applicant has already been taken")
```

Comment:
"has already been taken" is not appropriate here. Use locales to give this more appropriate text "Applicant is already an applicant on this submission" or something like that.